src/webutil/tableState.js

```diff
@@ -159,6 +159,7 @@ export function useTableState<T>(
     },
   }
 ): TableState<T> {
+  const data = useMemo(() => dataWrapper.data, [dataWrapper.data]);
```

Comment:
I like that we're memoizing inside TableState... does this obviate the need to do so outside before passing data in?

Reply:
Maybe but I think not. If I understand correctly, if dataWrapper.data is not memoized and is rebuilt every render, then everything downstream of it will also be rebuilt every render.